sled-agent/src/rack_setup/plan/service.rs

```diff
@@ -58,14 +58,23 @@ const OXIMETER_COUNT: usize = 1;
 // when Nexus provisions Clickhouse.
 // TODO(https://github.com/oxidecomputer/omicron/issues/4000): Use
 // omicron_common::policy::CLICKHOUSE_SERVER_REDUNDANCY once we enable
-// replicated ClickHouse
+// replicated ClickHouse.
+// Set to 0 when testing replicated ClickHouse.
```

Comment:
It seems like this is the number of _single-node_ ClickHouse servers. If that's right, can we make that clear with some documentation? 

I think it might be even better to have one single `const` here, to avoid confusion and accidentally starting something we didn't intend. For example:

```rust
enum ClickHouseSetup {
    Replicated { n_servers: usize, n_keepers: usize, },
    SingleNode { n_servers: usize },
}
// We're using single-node for now.
const CLICKHOUSE_SETUP = ClickHouseSetup::SingleNode { n_servers: 1 };

// If you want to test replication, uncomment this:
// const CLICKHOUSE_SETUP = ClickHouseSetup::Replicated { n_servers: 2, n_keepers: 3 };
```

Then later you can match on that type and figure out how many of each kind of zone to start.

Reply:
We still haven't 100% ruled out implementing [stage one](https://rfd.shared.oxide.computer/rfd/0468#_stage_1) of RFD 468. In this scenario we'll be wanting to have the ability to deploy both alongside each other. I'd like to keep these constants as they are unless we decide that we will not implement stage 1.

Reply:
Just want to add that these variables are going to go away very soon anyway as part of the reconfigurator work.